build.gradle.kts

```diff
@@ -130,9 +130,30 @@ buildTypes {
     }
 }
 
+val gradleMirrorUrl = getCiMirrorUrls()["gradle"]
+if (gradleMirrorUrl != null) {
```

Comment:
Please put this in a separate script and apply it to both the main build and buildSrc instead of copy-pasting.